cve_bin_tool/data_sources/osv_source.py

```diff
@@ -55,6 +55,7 @@ async def update_ecosystems(self):
             ecosystem = line.split("/")[-2]
             ecosystems.append(ecosystem)
 
+        ecosystems.remove("osv-vulnerabilities")
```

Comment:
```suggestion
        # Addresses https://github.com/intel/cve-bin-tool/issues/1930
        ecosystems.remove("osv-vulnerabilities")
```

Reply:
I think we need some explanation here as to why this one in particular needs to be excluded.  I'm not sure if referencing the github issue is the right way to go, so I'll just click the "request changes" button and let you think about whether there's something better to put in here.